Source/dafny.msbuild/DafnyCompileTask.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Data;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Task = Microsoft.Build.Utilities.Task;
+
+namespace DafnyMSBuild
+{
+    /**
+     * A custom task for verifying Dafny source code. Mainly implemented in C#
+     * so we can verify in parallel (by leveraging Parallel LINQ, which automatically
+     * leverages all available cores).
+     */
+    public class DafnyCompileTask : Task
```

Comment:
You shouldn't need to define this in the same way that we have the verification task. The latter was defined as a custom task to support parallel verification, and I don't think we need parallel compilation just yet.

To simplify this a whole bunch, I think you just need to add an extra parameter to the existing `<Exec ...>` line below...